packages/office-ui-fabric-react/src/components/GroupedList/examples/GroupedList.CustomCheckbox.Example.tsx

```diff
@@ -2,33 +2,30 @@ import * as React from 'react';
 import {
   GroupHeader,
   GroupedList,
-  IGroup,
   IGroupHeaderCheckboxProps,
   IGroupHeaderProps,
 } from 'office-ui-fabric-react/lib/GroupedList';
 import { IColumn, DetailsRow } from 'office-ui-fabric-react/lib/DetailsList';
 import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
 import { Selection, SelectionMode, SelectionZone } from 'office-ui-fabric-react/lib/Selection';
 import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
-
+import { useConst } from '@uifabric/react-hooks';
 import { createListItems, createGroups, IExampleItem } from '@uifabric/example-data';
 
 const groupCount = 3;
 const groupDepth = 1;
 
-export interface IGroupedListCustomCheckboxExampleState {}
-
-export class GroupedListCustomCheckboxExample extends React.Component<{}, IGroupedListCustomCheckboxExampleState> {
-  private _items: IExampleItem[];
-  private _columns: IColumn[];
-  private _groups: IGroup[];
-  private _selection: Selection;
+const onRenderHeader = (props: IGroupHeaderProps): JSX.Element => (
+  <GroupHeader onRenderGroupHeaderCheckbox={onRenderGroupHeaderCheckbox} {...props} />
+);
 
-  constructor(props: {}) {
-    super(props);
+const onRenderGroupHeaderCheckbox = (props: IGroupHeaderCheckboxProps) => <Toggle checked={props.checked} />;
 
-    this._items = createListItems(Math.pow(groupCount, groupDepth + 1));
-    this._columns = Object.keys(this._items[0])
+export const GroupedListCustomCheckboxExample: React.FunctionComponent = () => {
+  const items = useConst(createListItems(Math.pow(groupCount, groupDepth + 1)));
+  const groups = useConst(createGroups(groupCount, groupDepth, 0, groupCount));
+  const columns = useConst(
```

Comment:
This should use an initializer function too
```suggestion
  const columns = useConst(() =>
```